### PR TITLE
adding panaroo version 1.5.0

### PR DESCRIPTION
**Pull request by erinyoung**
There's a new version of panaroo! More information can be found in the [release notes](https://github.com/gtonkinhill/panaroo/releases/tag/v1.4.2), but basically genes can have really long names now.

I copied the prior Dockerfile and updated the software version ARG.

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [X] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by erinyoung** [comment]
There's a new version of panaroo!

More details here: https://github.com/gtonkinhill/panaroo/releases/tag/v1.5.0

As such, I've updated the files to the most recent version.

File: panaroo/1.5.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+LABEL base.image="mambaorg/micromamba:1.5.7"
+LABEL dockerfile.version="1"
+LABEL software="Panaroo"
+LABEL software.version=${PANAROO_SOFTWARE_VERSION}
+LABEL description="An updated pipeline for pangenome investigation"
+LABEL website="https://github.com/gtonkinhill/panaroo"
+LABEL license="https://github.com/gtonkinhill/panaroo/blob/master/LICENSE"
+LABEL maintainer="Kevin Libuit"
+LABEL maintainer.email="kevin.libuit@theiagen.com"
```
Comment (kapsakcj) [inline panaroo/1.5.0/Dockerfile:21]: Want to change lines 20 & 21 to yourself? or add `maintainer2` LABELs below?

**Comment by kapsakcj** [comment]
dockerfile looks good, and looks like the tests passed successfully. I can resolve the readme conflicts now

**Comment by kapsakcj** [comment]
I approve, just let me know about the maintainer LABEL and I can merge and deploy this one

File: panaroo/1.5.0/Dockerfile (hunk 1 above)
Comment (erinyoung) [reply panaroo/1.5.0/Dockerfile:21]: Oh yeah! I added myself as maintainer2

**Review by kapsakcj — APPROVED** [review]

**Pull request merged**